styles/z-bolt/style.css

```diff
@@ -145,8 +145,9 @@ entry {
 
 button.keyboard {
 	font-size: 30px;
-	margin: 0px;
-	/* margin-bottom: 10px; */
+	margin: 2px;
+	background-color: #538fbe;
```

Comment:
I'm going to leave this as-as for now (the background color) and merge this PR, but please be aware that OctoScreen has a "white on black" style to it, and this deviates from that.  I think here it's OK b/c the Connection panel is seldom used and is kind of a "one-off" feature.

Reply:
I'm sorry.  Truth be told, I just copied and pasted the style from another project, initially for testing, and it worked and I kept it.  That was bad form and I will try to adhere to the style of the app next time.